sympy/series/gruntz.py

```diff
@@ -277,7 +277,7 @@ def mrv(e, x):
         if e1.has(x):
             base_lim = limitinf(b1, x)
             exp_lim = limitinf(e1, x)
-            if base_lim is S.One and exp_lim is S.Infinity:
+            if base_lim is S.One and exp_lim.is_infinite is False:
```

Comment:
Isn't this different from the original logic for +oo?

Reply:
That was just changed in #25862. I suspect that it worked for that issue only because it allowed the -oo to pass to the line outside the if-block. When the test expression is inverted (see added test) a +oo needs to hit the expression outside of the if-block. So now both expressions are giving the correct result.